.github/workflows/unit-tests-php.yml

```
@@ -50,9 +47,10 @@ jobs:
                     composer require "composer/composer:~${{ matrix.composer }}.0" --dev --no-update
 
             -   name: Install dependencies
-                uses: ramsey/composer-install@v2
+                uses: ramsey/composer-install@v3
                 with:
                     dependency-versions: ${{ matrix.dependency-versions }}
+                    tools: composer:v${{ matrix.composer-ver }}
```

This is just `composer`, right?

```suggestion
                    tools: composer:v${{ matrix.composer }}
```

---

Good catch! 💪🏻 Actually it is `php-ver` but for composer it is just `composer`. Shoudl we rename it to `composer-ver` or should we rename `php-ver` to `php` only?

---

Renamed to composer-ver, thanks